reporting.md

```diff
@@ -61,8 +61,51 @@ The user account that runs the Power BI installer will be the administrator of t
 ## Backfill Sync Integrations
 The backfill job uploads historical data to Power BI and keeps the reports up to date.
 
-## Simeon Sync Power BI dataset
-Each row in the data source represents a single property of a configuration.
+## Baseline and Compliance report
+<br />
+<img src="https://raw.githubusercontent.com/simeoncloud/docs/master/assets/images/power_bi.png" width: 700; height: auto;/>
+
+*This screenshot shows the baseline compliance report for a demo environment.*
+
+The Baseline Compliance Report displays how each of your tenants compare to their baseline. You can see the comparison **by tenant**, **type of configuration**, and **configuration**. You can also drill down to see how properties in a **specific configuration** compare to the baseline.
+
+The data in the Baseline and Compliance report uses Power BI [import connections](https://learn.microsoft.com/en-us/power-bi/connect-data/desktop-directquery-about#import-connections) and is refreshed every 3 hours, which is the max number of automated refreshes allowed in Power BI Pro. You can refresh the report manually when viewing in Power BI. To do so:
+- Go to [Power BI](https://app.powerbi.com) > **Workspaces** > **Simeon Cloud**
+- Hover your mouse over the **Baseline Compliance Report** dataset > Click on the **refresh** icon
+
+## Building custom reports and alerts
+Simeon Cloud's integration of Power BI Reporting with Log Analytics provides significant opportunities to monitor tenants using custom reports and targeted alerts. The following resources are designed to help Simeon users interested in developing custom reports and alerts get started.
+
+### Custom Log Analytics queries
+Log Analytics allows you to develop specific queries using the data in your Log Analytics workspace. With Simeon Cloud, you can use the [Simeon Sync dataset](https://simeoncloud.github.io/docs/#/reporting?id=simeon-sync-power-bi-dataset) to tailor queries around nearly all aspects of your tenant configurations. To understand how Log Analytics works and how to start building custom queries, see [this guide](https://learn.microsoft.com/en-us/azure/azure-monitor/logs/log-analytics-tutorial).
+
+#### Sample Log Analytics queries
+The following custom queries were developed using the [Simeon Sync dataset](https://simeoncloud.github.io/docs/#/reporting?id=simeon-sync-power-bi-dataset) and highlight what is possible with Log Analytics integration within Simeon Cloud. Users are encouraged to develop custom queries to target specific resources and conditions of interest.
+
+*A configuration type is added/removed/changed in the portal and exported by Simeon Sync*
+```
+SyncLogs_CL
+| where Change_Type in ('Added', 'Removed', 'Changed') and Configuration_Type in ('MSGraph:Groups')
```

Comment:
We should note that MSGraph:Groups is just an example and how they can find the correct path for other configs. 

Microsoft’s API with colons between the path. In Simeon, this can be found in the parenthesis of the Sync report: Azure AD > Groups (MSGraph:Groups)

You also might want to update MSGraph:Groups to something more useful, like MSGraph:ConditionalAccess:Policies or something else that could be important